apps/cmpnbrs.c

```diff
@@ -9,7 +9,7 @@
 \version \verbatim $Id: m2mnbrs.c 17699 2014-09-27 18:05:31Z karypis $ \endverbatim
 */
 
-#include <GKlib.h>
+#include "GKlib.h"
```

Comment:
Why the change? I think the old form was correct. You use quotation marks for relative paths to the current file and angle brackets for files from the include path.

Reply:
The [`#include` documentation](https://gcc.gnu.org/onlinedocs/cpp/Include-Syntax.html) for GCC states:
> Both user and system header files are included using the preprocessing directive ‘#include’. It has two variants:
>
> `#include <file>`
>
> This variant is used for system header files. It searches for a file named *file* in a standard list of system directories. You can prepend directories to this list with the `-I` option (see Invocation).
>
> `#include "file"`
>
> This variant is used for header files of your own program. It searches for a file named *file* first in the directory containing the current file, then in the quote directories and then the same directories used for *\<file\>*. You can prepend directories to the list of quote directories with the `-iquote` option. 

The [Microsoft docs](https://docs.microsoft.com/en-us/cpp/preprocessor/hash-include-directive-c-cpp) say something similar.

What distinguishes the two syntax forms is that the quote form first checks the directory of the *current file*, then falls back to the angle bracket form if nothing is found. With this behavior in mind, I had used the quote form to distinguish headers that were part of the project, despite the fact that the behavior was actually falling back to the angle bracket form for any files in the `src` and `apps` directories. My rationale is that this allows the developer to quickly identify headers distributed with the project versus those provided by the system.

With that in mind, one could argue that in `src` and `apps`, these should be using angle bracket form.

Reply:
Thanks for quoting the specs. I was arguing like you said in the last sentence and I don't see an advantage in changing it. I wouldn't change it, but it probably is a matter of style.

Reply:
I am okay with either. Since it is not my project, I will defer to the primary maintainers if they have a preference. I will gladly change it back to using `<>` where appropriate.